### ISSUE TITLE
Charts and graphs : line wrapping on long labels

### ISSUE THREAD

**Issue by lucas-hay** [wet-boew/wet-boew#7994]
This issue was brought to my attention. When a label has a longer string of text, it does not wrap properly and is unreadable as can be seen below (scroll down to bar graph:  "Distribution of the Findings and Recommendations (F&R) by category of grievance for the period between January 1, 2017 and March 31, 2017"):
[https://www.canada.ca/en/military-grievances-external-review/services/mgerc-publications/ebulletin/ebulletin-april-2017.html](https://www.canada.ca/en/military-grievances-external-review/services/mgerc-publications/ebulletin/ebulletin-april-2017.html)

Is it possible to make these labels wrap?

**Comment by duboisp** [comment]
No, that is not possible currently. May be there is a [flot plugin](http://www.flotcharts.org/) that we can use to achieve it, I didn't look for it. 

However, it is possible to simplify the labeling as demo'ed in the [Labels and reference values](http://wet-boew.github.io/wet-boew/demos/charts/labelsandreferencevalue-en.html) working example by using a complex table.

**Comment by lucas-hay** [comment]
Thanks @duboisp, I will pass that on to the client who brought up the issue.

**Comment by shawnthompson** [comment]
Why would they have the word recommended in every label and in the title of the graph? Remove that word and problem solved!

**Comment by trtustian** [comment]
Mario and I were looking at this briefly. Maybe you can rotate the label text (in css) to solve the problem.
See the stackoverflow solution.
http://stackoverflow.com/questions/18634594/flot-bar-chart-xaxis-label-with-rotated-text-by-90-alignment-issue
